api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClientImpl.java

```diff
@@ -738,4 +739,21 @@ public int deleteUserAppsAsService(String userEmail) {
 
     return results.size();
   }
+
+  @Override
+  public Boolean deleteAllResources(String googleProject, boolean deleteDisk) {
+    return leonardoRetryHandler.run(
+        (context) -> {
+          try {
+            resourcesApiProvider.get().deleteAllResources(googleProject, deleteDisk);
+            return true;
+          } catch (ApiException e) {
+            log.log(
+                Level.SEVERE,
+                String.format("failed to delete all resources for project %s", googleProject),
+                e);
+            return false;
```

Comment:
swallow exception here is not a good practice, it may cause the upstream callers (it may have multiple callers).
We should let exceptions propagate to caller, so they can decide how to handle them. 
[go/java-practices/exceptions#dont_catch](go/java-practices/exceptions#dont_catch)